contrib/devtools/gen-manpages.sh

```diff
@@ -21,11 +22,19 @@ read -r -a ZECCOMMIT <<< "$(echo $ZECVERSTR | awk -F- '{ print $NF }')"
 # but has different outcomes for zcash-cli.
 echo "[COPYRIGHT]" > footer.h2m
 $ZCASHD --version | sed -n '1!p' >> footer.h2m
+grep -v "Bitcoin Core Developers" footer.h2m >footer-no-bitcoin-copyright.h2m
 
-for cmd in $ZCASHD $ZCASHCLI $ZCASHTX; do
+for cmd in $ZCASHD $ZCASHD_WALLET_TOOL $ZCASHCLI $ZCASHTX; do
```

Comment:
```suggestion
for cmd in $ZCASHD $ZCASHCLI $ZCASHTX; do
```

Reply:
Huh, I was sure I'd made this change already. Maybe I didn't commit it.